js/purify-3.1.2.js

```diff
@@ -0,0 +1,3 @@
+//# sourceMappingURL=purify.min.js.map
```

Comment:
I always remove these lines at the end, since it will cause a browser console error if those maps are not available but referred to. They would only be useful for debugging into this library, which we don't need to do, so we have always omitted these maps to reduce the transmitted files & sizes.

```suggestion
```

Reply:
In my testing actually no browser console error was shown, so maybe browser behavior has changed?